web_page/animation/script.js

```diff
@@ -0,0 +1,29 @@
+let idx = 1;
+
+show(idx);
+
+function next() {
+    show(idx+=1);
+}
+
+function previous() {
+    show(idx-=1);
+}
+
+function show(n) {
+    let cur = document.getElementsByClassName("el");
```

Comment:
```suggestion
    const cur = document.getElementsByClassName("el");
```

в рамках этого блока переменная не меняется, верно?

Reply:
Поменяла